logins-sql/src/util.rs

```diff
@@ -0,0 +1,137 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use error::*;
+use rusqlite::{types::{self, ToSql}, Row};
+use std::{fmt, time};
+use db::MAX_VARIABLE_NUMBER;
+use url::Url;
+
+pub fn each_chunk_mapped<'a, T: 'a>(
```

Comment:
It's odd to call a function `map` when you're not returning a result.  Oh, now I see -- it's mapping the items into `&ToSql`.  You really need this?  OK then.

Reply:
Yep, it's really annoying. TBH I'm not a fan of the ToSql API offered by rusqlite, IMO an enum-based API would have been both more usable and more performant.